pam.c

```diff
@@ -26,6 +28,15 @@
 
 static pam_handle_t *pam_handle;
 
+static void change_identity (struct passwd *pw) {
+	if (initgroups(pw->pw_name, pw->pw_gid) == -1)
+		exit(1);
```

Comment:
Are you sure exit(1) is the right thing to do?
If this isn't being run through systemd won't it just cause the graphical login system to exit?

Reply:
It's a massive security risk to continue if you fail to unset the `root` group from the running user, especially if they're not supposed to have it. 

However, this WILL NOT exit the login manager, as it gets called inside of a `fork()`ed process. As such, I've updated it to the more correct `_Exit()` call instead,

Sorry about the spacing, I think I've fixed it now.

Reply:
....I mean now! Now spacing should be fixed.